pkg/metaserver/util/util.go

```diff
@@ -138,7 +138,7 @@ func GetMessageAPIVersion(msg *beehiveModel.Message) string {
 func GetMessageResourceType(msg *beehiveModel.Message) string {
 	obj, ok := msg.Content.(runtime.Object)
 	if ok {
-		return UnsafeKindToResource(obj.GetObjectKind().GroupVersionKind().Kind)
+		return obj.GetObjectKind().GroupVersionKind().Kind
```

Comment:
the above fix is to fix msg resourcetype, from cloud to edge, from singular to plural.
But is this fixed line related to `res type not found` error?